### ISSUE TITLE
Minor styling error with title-only admonition blocks

### ISSUE THREAD

**Issue by ZaLiTHkA** [squidfunk/mkdocs-material#616]
## Description

When using admonition as only the title and no body..

    !!! info "hello world"

..there's some bottom padding that pushes out the wrapping container a bit, leaving a white bar along the bottom of the coloured section of the admonition block.

### Expected behavior

I expect everything to be pixel-perfect, with no unwanted padding or blurry corners! Lol. 🖖 

### Actual behavior

As per screenshot:

![2017-11-27 15_04_24-without-fix](https://user-images.githubusercontent.com/2149312/33268110-9f263622-d384-11e7-9460-ab06d943a715.png)

### Steps to reproduce the bug

>n/a

### Package versions

* Python: `2.7.13`
* MkDocs: `0.17.1`
* Material: `2.2.0`

### Project configuration

>n/a

### System information

* OS: Windows 10 x64
* Browser: Google Chrome 62.0.3202.94

**Comment by ZaLiTHkA** [comment]
Not sure if this is the *most* elegant solution, there could be better ways to handle it, but in the meantime I'm getting around this with the following custom CSS:

```css
.md-typeset .admonition, .md-typeset details {
  padding-bottom: 0;
}

html .md-typeset .admonition :last-child, html .md-typeset details :last-child {
  padding-bottom: 1rem;
}
```

**Comment by ZaLiTHkA** [comment]
Not sure if this is the *most* elegant solution, there could be better ways to handle it, but in the meantime I'm getting around this with the following custom CSS:

```css
.md-typeset .admonition, .md-typeset details {
  padding-bottom: 0;
}

html .md-typeset .admonition :last-child, html .md-typeset details :last-child {
  padding-bottom: 1rem;
}
```

**Comment by squidfunk** [comment]
Fixed in #621.

**Comment by squidfunk** [comment]
Released as 2.2.1.

**Comment by ZaLiTHkA** [comment]
Thanks for the fix. :)
